README.md

```diff
@@ -148,17 +148,27 @@ to attempt to force one or more of us to tamper with the software.
 Create a new API key:
 
 ```sh
-$ turnkey gen --name my-test-key
+$ turnkey gen --organization $ORGANIZATION_ID
 {
-    "privateKeyFile": "/Users/rno/.config/turnkey/keys/my-test-key.private",
-    "publicKeyFile": "/Users/rno/.config/turnkey/keys/my-test-key.public"
+   "privateKeyFile": "/Users/andrew/Library/Application Support/turnkey/keys/default.private",
+   "publicKey": "0236f17892a4649d97b2e4a4ad3c22d815e4e77848a0b8e4a5b0956ae4d6be382e",
+   "publicKeyFile": "/Users/andrew/Library/Application Support/turnkey/keys/default.public"
 }
 ```
 
-Make an API request:
+Make an API request (using the default API key created above):
 
 ```sh
-$ turnkey request --host coordinator-beta.turnkey.io --path /api/v1/sign --body '{"payload": "hello from TKHQ"}' --key=my-test-key
+$ turnkey request --host coordinator-beta.turnkey.io --path /api/v1/sign --body '{"payload": "hello from TKHQ"}'
```

Comment:
Here, too, `coordinator-beta.turnkey.io` is the default; no need to list it explicitly in the doc here.

Reply:
good call! removed.